### PR TITLE
Fix/tfn 757 dac remediations

### PR DESCRIPTION
**Pull request by ljones92**
# Description

-   Address all high priority DAC issues apart from table issues

# Testing instructions

- Ensure all journeys and error states work as expected

# Type of change

-   [ ] feat - A new feature
-   [x] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

# Checklist:

-   [x] Able to run pr locally
-   [x] Followed acceptance criteria
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [ ] I have made corresponding changes to the documentation if applicable
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/pages/productDetails.tsx [hunk 1]
```diff
@@ -71,61 +75,61 @@ const ProductDetails = ({
...
-                                                aria-label="Enter amount in pounds"
-                                                name="productDetailsPriceInput"
-                                                data-non-numeric
-                                                type="text"
-                                                id="product-details-price"
-                                                defaultValue={productPrice || ''}
-                                            />
-                                        </FormElementWrapper>
+                            <FormGroupWrapper errors={errors} errorId="product-name-error">
+                                <>
```
Comment (dannydaviesIW) [inline src/pages/productDetails.tsx:79]: This page no longer renders the error bar alongside the input fields

File: src/pages/productDetails.tsx [hunk 2]
```diff
@@ -71,61 +75,61 @@ const ProductDetails = ({
...
+                                                errorClass="govuk-input--error"
+                                            >
+                                                <input
+                                                    className="govuk-input govuk-input--width-10 govuk-currency-input__inner__input"
+                                                    name="productDetailsPriceInput"
+                                                    data-non-numeric
+                                                    type="text"
+                                                    id="product-details-price"
+                                                    defaultValue={productPrice || ''}
+                                                />
```
[the remark this answers is not in the document]
Comment (ljones92) [reply src/pages/productDetails.tsx:126]: I've given these hints a bit of an overhaul across this and the multiple products page, nice spot 👍

File: src/pages/api/apiUtils/userData.ts [hunk 3]
```diff
@@ -55,6 +57,31 @@ import { isFareZoneAttributeWithErrors } from '../../csvZoneUpload';
...
+                description: parsedEntry.description,
+                purchaseLocations: parsedEntry.purchaseLocations,
+                paymentMethods: parsedEntry.paymentMethods,
+                ticketFormats: parsedEntry.ticketFormats,
+            };
+            salesOfferPackageList.push(formattedPackageObject);
+        });
+
+    return salesOfferPackageList;
+};
```
[the remark this answers is not in the document]
Comment (ljones92) [reply src/pages/api/apiUtils/userData.ts:83]: Yeah, this was addressed by making the page title the label

File: src/pages/groupSize.tsx [hunk 4]
```diff
@@ -28,34 +28,33 @@ const GroupSize = ({ groupTicketInfo, csrfToken }: GroupSizeProps & CustomAppPro
...
+                            errorId={errors.length > 0 ? errors[0].id : ''}
+                            errorClass="govuk-input--error"
+                        >
+                            <input
+                                className="govuk-input govuk-input--width-2"
+                                id="max-group-size"
+                                name="maxGroupSize"
+                                type="text"
+                                defaultValue={groupTicketInfo.maxGroupSize}
+                            />
```
[the remark this answers is not in the document]
Comment (ljones92) [reply src/pages/groupSize.tsx:56]: Yeah, this was addressed by making the page title the label

**Review by dannydaviesIW — APPROVED** [review]

**Review by dannydaviesIW — APPROVED** [review]

**Pull request merged**